src/TcoInspectors/src/XAE/XAE/TcoInspectors/POUs/TcoInspector/TcoInspector.TcPOU

```diff
@@ -176,7 +176,7 @@ END_VAR
 ]]></Declaration>
       <Implementation>
         <ST><![CDATA[//// Hate to do this guys but I get compile error, seems to be a bug in tc
```

Comment:
😄

Reply:
🤣 I will leave it there as a testimony to the history😁.